tests/data/framework_processor_data/evaluate.py

```diff
@@ -0,0 +1,5 @@
+"""
+Integ test file evaluate.py
```

Comment:
Nit: As these test data are used by pipeline tests only. Let's move it to the `data/pipeline` or `data/workflow` folder.
In the future we'd better merge these 2 folders into 1